hack/BUILD.bazel

```diff
@@ -0,0 +1,92 @@
+package(default_visibility = ["//visibility:public"])
+
+test_suite(
+    name = "verify-all",
+    tags = ["lint"],  # picks up all non-manual targets with this tag
+)
+
+GAZELLE = "@bazel_gazelle//cmd/gazelle"
+
+KAZEL = "@io_k8s_repo_infra//cmd/kazel"
+
+GO = "@go_sdk//:bin/go"
+
+sh_binary(
+    name = "update-deps",
+    srcs = ["update-deps.sh"],
+    args = [
+        "$(location %s)" % GO,
+        "$(location %s)" % GAZELLE,
+        "$(location %s)" % KAZEL,
+        "$(location :update-bazel)",
```

Comment:
nit: in other scripts that depend on other scripts (in test-infra), there seems to be a pattern of structuring args so that it's possible to `shift` the arguments the current script needs off the arg stack, and then calling "$@" to call the next dependent script. This would mean that the `update-bazel` arg/location would go *after* the GO argument, but before gazelle and kazel.

When copying/borrowing bits from test-infra, I really liked this pattern as it made it easy to just copy and paste args from update-* into a verify-* or other update-* script, and get on 😄 not sure how important it is though